adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,75 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_4_1']
+=== Release 1.4.1, July 2019
+
+[id='sec.1_4_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.17.1.
+* Added option to deploy UAA from the SCF Helm chart. In new CAP deployments that do not require a separate UAA deployment, setting `enable.uaa=true` in the Helm values automatically creates a UAA StatefulSet within the SCF deployment. This removes the manual step of copying the UAA_CA_CERT during setup. In this configuration UAA uses the same database as SCF, reducing the memory footprint of the cluster. This option should be left as `false` (default) for upgrades of existing CAP clusters.
+
+[id='sec.1_4_1.feature']
+==== Features and Fixes
+
+* Set the default value of `AZ_LABEL_NAME` to `failure-domain.beta.kubernetes.io/zone`
+* Simplified service accounts and pod security policies
+* Switched to log-cache for container metrics
+* Implemented a patch to squash Cloud Controller database migrations
+* Fixed version and SHA1 of `cf-mysql-release` tied to version 36.15.0
+* Fixed TLS issues in `log-cache`
+
+* Includes these {cf} component versions:
+
+** app-autoscaler: 1.2.1
+** bits-service: 2.26.0
+** bpm: 1.0.0
+** capi: 1.79.0
+** cats: 7.11
+** cf-deployment: 7.11
+** cf-mysql: 36.15.0
+** cf-routing: 0.187.0
+** cf-sle12: 1.75.11
+** cf-smoke-tests: 40.0.51
+** cf-syslog-drain: 10.0
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.281.0
+** cflinuxfs3: 0.108.0
+** credhub: 2.1.2
+** diego: 2.30.0
+** eirini: 0.0.4
+** garden-runc: 1.19.1
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 105.2
+** loggregator-agent: 3.9
+** nats: 26
+** nfs-volume: 1.5.2
+** postgres-release: 26
+** scf-helper: 1.0.2
+** statsd-injector: 1.9.0
+** uaa: 68.0
+* Buildpacks:
+** binary-buildpack: 1.0.32
+** dotnet-core-buildpack: 2.2.12
+** go-buildpack: 1.8.41
+** java-buildpack: 4.19.1 
+** nginx-buildpack: 1.0.14
+** nodejs-buildpack: 1.6.51
+** php-buildpack: 4.3.77
+** python-buildpack: 1.6.34
+** ruby-buildpack: 1.7.40
+** staticfile-buildpack: 1.4.43
+
+[id='sec.1_4_1.issue']
+==== Known Issues
+
+* `cf-deployment` 7.11 is the final Cloud Foundry version that supports the `cflinuxfs2` stack. The `cflinuxfs2` and `sle12` stacks are deprecated in favor of `cflinuxfs3` and `sle15` respectively. Start planning to migrate applications to those stacks for futureproofing, as these stacks will be removed in a future release. The Stack Auditor plugin for `cf` can help with this migration (see https://docs.cloudfoundry.org/adminguide/stack-auditor.html). 
+
+* Do not use the new `enable.uaa=true` SCF chart setting in upgrades of existing CAP 1.4.0 clusters. This will cause a new UAA role to be created which SCF will use instead of the existing UAA deployment. This option is disabled by default in the chart values. 
```

Comment:
I am a bit confused: This bullet first tells me there is an upgrade issue. In the second sentence, it tells me what this thing does.
The functionality is actually also explained in the "New" section of the document, so maybe a reference back would suffice:
```
* When upgrading CAP 1.4.1 clusters, do not enable the new `enable.uaa=true` SCF chart setting (see also <<sec.1_4_1.new>>). [Insert a sentence about the catastrophic consequences.]
```

Reply:
The wording that ended up going higher started here. It's not an upgrade issue per se, but a change in the default setup, hence why we highlighted it in the What's New section. The docs will cover this in more detail but I've added a preface to the earlier section.